pipelines/tikv/pd/latest/pod-pull_integration_realcluster_test.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.21:latest"
+      tty: true
+      resources:
+        requests:
+          memory: 12Gi
+          cpu: "4"
```

It is recommended to set the values of "request" and "limit" to be the same.
```suggestion
          memory: 16Gi
          cpu: "6"
```